.github/workflows/pull.yml

```diff
@@ -10,7 +10,7 @@ jobs:
     steps:
       - name: Populate Dafny versions list
         id: populate-dafny-versions-list
-        run: echo "dafny-versions-list=['4.2.0', '4.4.0', '4.8.0']" >> $GITHUB_OUTPUT
+        run: echo "dafny-versions-list=['4.8.0']" >> $GITHUB_OUTPUT
```

Comment:
If we're claiming 4.5 and higher works, we should still test on 4.5. If not then get_dafny_version.sh should just assert the version is 4.8 or higher.